### PR TITLE
editorial change: referring to dcat vocabulary specification

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
This PR is related to issue #430, and it is solving only the doubt raised by Antoine.

### CODE REVIEW

**Review by pwin — APPROVED** [review]
fine with this

**Review by davebrowning — COMMENTED** [review]
(User having problems with github suggestions so afraid we'll have to do it the hard way)

Looking at this in context, I'm less sure if it's enough.  Where we use an externally defined term we state the external definition has precedence.  If that changes (non-backward compatibly) then there is a risk that it breaks something that was DCAT compliant.  A change to the text at line 58 so that it says "...summary definitions of the externally-defined terms _in the DCAT vocabulary_..." helps a bit, but we could also add "...possible changes to _other_ external definitions..." to line 60.

That then makes sense to me -

**Comment by riccardoAlbertoni** [comment]
> Some suggested tweaks....

@davebrowning : I am afraid I am unable to see the suggestions. Do you have already posted them, or they are still under preparation?

**Comment by davebrowning** [comment]
> > Some suggested tweaks....
> 
> @davebrowning : I am afraid I am unable to see the suggestions. Do you have already posted them, or they are still under preparation?

Couldn't get it to work - added as a vanilla comment.  Apologies.  Hope it makes sense

File: dcat/index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h3 id="external_terms">External terms</h3>
     <p>DCAT incorporates terms from pre-existing vocabularies where stable terms with appropriate meanings could be found, such as <a href="http://xmlns.com/foaf/0.1/homepage"><code>foaf:homepage</code></a> and <a href="http://purl.org/dc/terms/title"><code>dct:title</code></a>.
       Informal summary definitions of the externally-defined terms are included here for convenience, while authoritative definitions are available in the normative references.
```
Comment (riccardoAlbertoni) [inline dcat/index.html]: Suggestion by @davebrowning 
```suggestion
      Informal summary definitions of the externally-defined terms are included in the DCAT vocabulary for convenience, while authoritative definitions are available in the normative references.
```

Comment (davebrowning) [reply dcat/index.html]: Thanks - that is what I was trying to do...

File: dcat/index.html [hunk 2]
```diff
@@ -57,7 +57,7 @@ <h3 id="external_terms">External terms</h3>
...
       Changes to definitions in the references, if any, supersede the summaries given in this specification.
-      Note that conformance to DCAT (<a href="#conformance"></a>) concerns usage of only the terms in the DCAT namespace itself, so possible changes to the external definitions will not affect the conformance of DCAT implementations.</p>
+      Note that conformance to DCAT (<a href="#conformance"></a>) concerns usage of only the terms in the DCAT vocabulary specification, so possible changes to the external definitions will not affect the conformance of DCAT implementations.</p>
```
Comment (riccardoAlbertoni) [inline dcat/index.html]: Suggestion by @davebrowning 
```suggestion
      Note that conformance to DCAT (<a href="#conformance"></a>) concerns usage of only the terms in the DCAT vocabulary specification, so possible changes to other external definitions will not affect the conformance of DCAT implementations.</p>
```

**Review by davebrowning — APPROVED** [review]

**Comment by riccardoAlbertoni** [comment]
> > > Some suggested tweaks....
> > 
> > 
> > @davebrowning : I am afraid I am unable to see the suggestions. Do you have already posted them, or they are still under preparation?
> 
> Couldn't get it to work - added as a vanilla comment. Apologies. Hope it makes sense

No problem at all, please check if I have implemented your suggestions correctly and in the case do not forget to approve the PR.

**Comment by dr-shorthair** [comment]
Fixes #430

**Pull request merged**